UPDATING.md

```diff
@@ -12,3 +12,5 @@
 `v0.6.0-alpha.0` cannot work on top of latest version.
 - Address (offset) of programs for Pico was changed from 0x100A0000 to 0x10100000. UF2
 binaries need to be rebuilt with the proper offset using `uf2tool`.
+- On ESP32, SSID and PSK stored in NVS are no longer read by network module. Applications
```

Comment:
I think we need to put this in a new Breaking Changes section.

Reply:
This information here is good and useful, but I would add a "Removed" section as specified in https://keepachangelog.com/en/1.1.0/
I would say something like: "- Removed support to ESP32 NVS from `network` module in order to make it generic. See also [UPDATING.md].